tests/unit/ui/components/TestShowMoreCard.js

```diff
@@ -40,7 +40,7 @@ describe(__filename, () => {
 
   it('truncates the contents if they are too long', () => {
     const root = render({ children: 'Hello I am description' });
-    root.truncateToMaxHeight({ clientHeight: 101 });
+    root.truncateToMaxHeight({ clientHeight: 146 });
```

Comment:
Could you export `MAX_HEIGHT` in the `ShowMoreCard` component and use it here? Like:

```
root.truncateToMaxHeight({ clientHeight: MAX_HEIGHT + 1 });
```

Reply:
This make things easy :)